docs/reference/search/rrf.asciidoc

```diff
@@ -82,10 +81,12 @@ GET example-index/_search
 ----
 // TEST[skip:example fragment]
 
-In the above example, we first execute the kNN search to get its global top 50 results.
-Then we execute the query to get its global top 50 results. Afterwards, on a coordinating
-node, we combine the knn search results with the query results and rank them based on the
-RRF method to get the final top 10 results.
+In the above example, we first execute the kNN search specified by the `knn` retriever to
```

Comment:
Would it make sense to annotate this explanation with numbers in the code snippet?

Reply:
I played with annotating certain lines for a bit, but I couldn't come up with something that I felt like improved the readability for the example.

Reply:
I changed this one as well. Thank you again.